TourMate/TourMate/Models/DataModels/FirebaseAdaptedAccommodation.swift

```diff
@@ -34,12 +34,13 @@ class FirebaseAdaptedAccommodation: FirebaseAdaptedPlan {
 
     required init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
+
+        address = try container.decode(String?.self, forKey: .address)
+        phone = try container.decode(String?.self, forKey: .phone)
+        website = try container.decode(String?.self, forKey: .website)
+
         let superDecoder = try container.superDecoder()
         try super.init(from: superDecoder)
```

Comment:
This would prevent the nesting
```suggestion
        try super.init(from: decoder)
```

Reply:
Actually the creation of base and super nesting, is it done in the encoder instead? Because if I'm not wrong that part creates the nested structure then passes it to store in Firebase?

Reply:
You are right, we need to change it at both the encoder and decoder part

Reply:
I will add it as an issue for sprint 2